source/tests/glkernel-test/sample_test.cpp

```diff
@@ -25,3 +25,13 @@ TEST_F(sample_test, poisson_square_compile)
 
     glkernel::sample::poisson_square(dkernel2);
 }
+
+TEST_F(sample_test, stratified_compile) {
+	auto fkernel2 = glkernel::kernel2{ 1 };
```

Comment:
start block on next line (embrace code style of the surrounding/pre-existing source)